commons/weather/weather_current.avsc

```diff
@@ -0,0 +1,32 @@
+{
+  "namespace": "org.radarcns.weather",
+  "type": "record",
+  "name": "WeatherCurrent",
+  "doc": "Weather information at the time reported.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "sunrise", "type": "double", "doc": "sunrise time in hours after midnight (h). The difference with sunset is the hours of daylight."},
```

Comment:
Why hours with type `double`, and not hours with type `integer` or seconds with type `integer`? Hours with a fraction are a bit unintuitive to evaluate (base 60 vs base 10).

Reply:
Changed to an integer amount of minutes. 
Pro's: 
- indicates that it is up to minutes precision
- clear distinction from UTC time (6 orders of magnitude smaller)
Cons: 
- more difficult to interpret than hours